<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Registration</class>
 <widget class="QMainWindow" name="Registration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="create_account_button">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>370</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Создать аккаунт</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="new_fio">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>120</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>100</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>120</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ФИО</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="new_phone_number">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>160</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>11</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>160</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Телефон</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="new_email">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>200</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>128</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>200</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Почта</string>
    </property>
   </widget>
   <widget class="QLabel" name="reg_birthdate_label">
    <property name="geometry">
     <rect>
      <x>198</x>
      <y>240</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Дата рождения</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="new_password">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>300</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>255</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>300</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Пароль</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="new_login">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>80</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>100</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>80</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Логин</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="new_birthdate">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>240</y>
      <width>110</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
